ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx

```diff
@@ -22,14 +24,23 @@ class UpdatesWizardDetailList extends Component {
             <Loader />
           </div>
         ) : (
-          _.map(selectedHardwares, item => (
-            <UpdatesWizardDetailListItem
-              key={item.name}
-              item={item}
-              wizardData={wizardData}
-              onStep2DataSelect={onStep2DataSelect}
-            />
-          ))
+          <>
+            {_.map(selectedHardwares, item => (
+              <UpdatesWizardDetailListItem
+                key={item.name}
+                item={item}
+                wizardData={wizardData}
+                onStep2DataSelect={onStep2DataSelect}
+              />
+            ))}
+            {selectedHardwares.length < 2 && (
```

Comment:
Change "selectedHardwares.length < 2" to "selectedHardwares.length === 1"

Reply:
ok